<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>338</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Game over</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 32, 32);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: qlineargradient(spread:pad, x0:1, y1:0, x2:1, y2:0, stop:0 rgb(60, 227, 227), stop:1 rgba(0, 0, 128, 255));
border-radius:15px</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="textLeaderboard">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>280</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40,40,40);

color: white;</string>
    </property>
    <property name="text">
     <string>GAME OVER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreText">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>TF2 Secondary</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">Border: 1px solid grey;
padding: 10px;
color: rgb(245, 245, 245);
background-color:rgb(152, 193, 255)</string>
    </property>
    <property name="text">
     <string>Score : 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="LevelText">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>TF2 Secondary</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">Border: 1px solid grey;
padding: 10px;
color: rgb(245, 245, 245);
background-color:rgb(152, 193, 255)</string>
    </property>
    <property name="text">
     <string>Level : 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimeText">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>TF2 Secondary</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">Border: 1px solid grey;
padding: 10px;
color: rgb(245, 245, 245);
background-color:rgb(152, 193, 255)</string>
    </property>
    <property name="text">
     <string>Time : 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>220</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>TF2</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0);
border: 0;
color: rgb(150, 150, 150);
}
QPushButton:hover{
background-color: rgba(255, 255, 255, 45);
color: rgb(255,255,255);
}
</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
